### DISCUSSION TITLE
Help: Where does this log come from?

### DISCUSSION THREAD

**Discussion by Booitt** [WhiskeySockets/Baileys#236] [General Questions]
I like to keep my console clean so I can easily see my custom logs, but there are some session logs popping up that I can't determine the source of.

The logs are like:

```
Closing stale open session for new outgoing prekey bundle
Closing session: SessionEntry ...
```
I would appreciate it if anyone could point out where these logs are coming from and/or how to stop them from logging.

**Comment by vinikjkkj** [comment]
same

**Comment by Prodigy90** [comment]
https://github.com/WhiskeySockets/libsignal-node/blob/83a3e3a3864511cb74df1b796373f0d49d071134/src/session_record.js#L273
